src/event_message/mod.rs

```diff
@@ -93,12 +92,40 @@ impl From<EventMessage> for TimestampedEventMessage {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+// KERI serializer should be used to serialize this
+#[derive(Debug, Clone, Deserialize)]
 pub struct SignedEventMessage {
     pub event_message: EventMessage,
+    #[serde(skip_serializing)]
+    pub payload_type: PayloadType, 
```

Comment:
Why we need `payload_type` field here? We have separate structures for signed receipts, so isn't it always `-A##` for `SignedEventMessage` with attached signatures?

Reply:
only if it is `ED25519` signature...
https://github.com/decentralized-identity/keri/blob/master/kids/kid0001.md#base64-master-code-table

Reply:
But signature type is already encoded in `AttachedSignaturePrefix` structure. I thought `payload_type` here is to encode the number of attached signatures with Counter Four Character Codes.

Reply:
it's there for encoding both type and size...